### PR TITLE
[ADD] sale_order_line_field_from_product_attribute: new module (was sale_attribute_to_field)

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

File: sale_attribute_to_field/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2023 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Store Attribute value sis Sales Order line fields",
+    "version": "16.0.1.0.0",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "summary": "Store Attribute value sis Sales Order line fields",
+    "website": "https://github.com/OCA/sale-workflow",
+    "license": "LGPL-3",
```
Comment (rousseldenis) [inline sale_attribute_to_field/__manifest__.py]: It seems your code is mixed LGPL/AGPL

**Review by rousseldenis — COMMENTED** [review]
I would have renamed to 'sale_order_line_field_from_product_attribute'

**Comment by dreispt** [comment]
Module name changed, license fixed.

**Review by ckolobow — APPROVED** [review]

**Pull request merged**